modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -291,21 +291,29 @@ In cases of high load with endpoints that do not support load balancing, applyin
 
 You can also choose to define the minimum number of machines that a cluster requires to remain in an operational state. This configuration grants you a consistency improvement in the overall performance.
 
-[[cluster_fips]]
-== Clustering in FIPS mode
+== Encrypt Communication Between Cluster Nodes
+
+You can encrypt traffic between cluster nodes by defining the `mule.cluster.network.encryption.key` property. You can define this property in the `wrapper.conf` file, or as a system property.
 
-To enable clustering in FIPS mode, Mule Runtime must run in FIPS mode. See xref:fips-140-2-compliance-support.adoc[FIPS Compliant support] for more information about this.
+When you define an encryption key, all communication between cluster nodes is encrypted using the defined key and the AES encryption algorithm.
+
+[[cluster_fips]]
+== Enable Clustering in FIPS Mode
 
-If you are running in FIPS mode, then a key for the cluster must be defined. You can define the key by setting the `mule.cluster.network.encryption.key` property (`wrapper.conf` or system property). If you are using a clustered environment in FIPS mode and not setting the encryption key, the application will fail to deploy with a message `Cluster key must be defined in FIPS mode.`. It is not necessary to run in FIPS mode to define the encryption key and encrypt all communications. But if the runtime is running in FIPS mode, then the encryption key must be defined.
+To enable clustering in FIPS mode, complete the following tasks:
 
-With the encryption key defined, all communications between cluster nodes will be encrypted using an AES encryption algorithm, with the defined key.
+* Run Mule runtime engine in FIPS mode. +
+See xref:fips-140-2-compliance-support.adoc[FIPS Compliant support] for configuration instructions.
+* Encrypt communication between cluster nodes. +
+Define the `mule.cluster.network.encryption.key` property, either in the `wrapper.conf` file or as a system property.
 
 === Warnings
 
-* The encryption key for all the clustered nodes must be the same for this feature to work.
-* Take into account that the use of a clustered environment in FIPS mode downgrades performance because all the communications between nodes will be encrypted (and decrypted).
+* If you don't define the encryption key, the application fails to deploy with message: `Cluster key must be defined in FIPS mode.`.
+* Use the same encryption key for all the clustered nodes, otherwise this feature won't work.
+* Running a clustered environment in FIPS mode causes a negative impact in performance, because all the communication between nodes will be encrypted (and decrypted).
```

Comment:
```suggestion
* Running a clustered environment in FIPS mode causes a negative impact in performance, because all the communication between nodes is encrypted (and decrypted).
```